components/uuf-core/src/main/java/org/wso2/carbon/uuf/core/API.java

```diff
@@ -192,6 +200,28 @@ public Session createSession(User user) {
     }
 
     /**
+     * Returns whether the current user has the given permission.
+     *
+     * @param permission permission to be checked
+     * @return {@code true} if the user has the given permission, otherwise {@code false}
+     */
+    public boolean hasPermission(Permission permission) {
+        Optional<Session> session = getSession();
+        if (!session.isPresent()) {
+            return false;
+        }
+        if (permission.equals(Permission.ANY_PERMISSION)) {
+            return true;
+        }
+        if (authorizer == null) {
+            LOGGER.warn("Authorizer is not configured and is null. This will result in all permission checks to " +
```

Comment:
Since we put a log in the app deployment time stating authorizer is null, there is no need for this log.